### DISCUSSION TITLE
v11 arr-see-three

### DISCUSSION THREAD

**Discussion by KevinJump** [KevinJump/uSync#429] [Announcements]
_This is a release candidate release of uSync for Umbraco 11 - it is for testing etc. not recommended on production sites_

The release candidate for Umbraco 11 (rc-3) is here. Don't worry you didn't miss RC1 and RC2 - we just didn't release them.

v11-rc3 is branched from the [v10.3 release of uSync](https://github.com/KevinJump/uSync/releases/tag/v10.3.0) so has all the same goodies. There are only a few minor differences for v11 (so far)

# Internal Updates
- Change of the root static assets folder from wwwroot to wwwroot/uSync (for neatness)
- removal of the package.manifest file (didn't get used anyway - caused unnecessary git changes)

# Non changes of note

## We still haven't changed the default folder for uSync from uSync/v9!
We didn't change this for v10 a move many (including myself), thought might have been a mistake, but given how little actually changed in terms of formatting etc, we left it at 9 so upgrades where easy - now v11 is here, we could right this wrong, 

but now I am not sure 🤔 - does changing this to v11 now actually confuse people even more, it's like we've all become use to the folder being called v9 even though we are on v11, shall we just leave it at v9, and we will come up with a better naming convention when we get to a meatier upgrade? If you have any thoughts, we would like to hear them. 

--- 

# To get uSync.

You will need to be running Umbraco 11-rc3  - https://our.umbraco.com/download/releases/1100

```
dotnet add package uSync --version 11.0.0-rc3
```

That is it. 

<hr /><em>This discussion was created from the release <a href='https://github.com/KevinJump/uSync/releases/tag/'>v11 arr-see-three</a>.</em>

**Comment by leekelleher** [comment]
`~/uSync/data/`? 🤔 Nooooooooooo! 😱 

The crux of the inner folder naming is that for Umbraco 7 it was called "data", then for Umbraco 8+ the uSync XML schema changed (for the better), but needed a way to differentiate between the schemas.

The "v9" folder did initially irk me when using Umbraco 10, but at the end of the day, it's just a folder.  For Umbraco 11, it could be renamed, but if the uSync XML schema isn't changing, then I'd be tempted to keep it with "v9". 🤷

**Comment by PhyxionNL** [comment]
> We still haven't changed the default folder for uSync from uSync/v9!

I would opt to remove the vX completely.

**Comment by mikaellindemann** [comment]
I would use the major part of a "uSync file format version" which is separate from the uSync/Umbraco version (I don't remember if such a version number already exists).

Then it's like a database schema.
Non-breaking changes can be done within the same major version.
Breaking changes can be migrated from one major version to the next (if desirable).

**Reply by KevinJump** [reply]
yeah good idea - 

We do have a 'file format version' - there have been no 'breaking' changes since v8, we use it now mainly to warn people, because when the format changes (event a little bit) you get false positives and slower reports.
